etc/errors/anvil.json

```diff
@@ -780,6 +780,321 @@
+                },
+                {
+                  "name": "tx_gas_per_pubdata_limit",
+                  "type": "U256"
+                },
+                {
+                  "name": "max_gas",
+                  "type": "U256"
+                }
+              ],
+              "doc": {
+                "summary": "Transaction validation failed due to excessive gas per pubdata limit.",
+                "description": [
+                  "This error occurs when a transaction's gas per pubdata limit exceeds the maximum allowed gas.",
+                  "Anvil-zksync enforces gas limits that simulate those of the real ZKSync network to ensure testing accuracy.",
+                  "Note: When anvil-zksync is in forking mode, it lock-in and uses gas price at the forked block"
+                ],
+                "likely_causes": [
+                  {
+                    "cause": "The transaction's gas per pubdata limit is set too high.",
+                    "fixes": [
+                      "Decrease the gas per pubdata limit"
+                    ]
+                  }
+                ]
+              }
+            },
+            {
+              "name": "TransactionValidationFailedMaxFeePerGasTooLow",
+              "code": 3,
+              "message": "Transaction {transaction_hash} allows paying a maximum of {max_fee_per_gas} fee per gar, but the current L2 gas price is {l2_gas_price}, which makes it too expensive.",
+              "fields": [
+                {
+                  "name": "max_fee_per_gas",
+                  "type": "U256"
+                },
+                {
+                  "name": "transaction_hash",
+                  "type": "H256"
+                },
+                {
+                  "name": "l2_gas_price",
+                  "type": "U256"
+                }
+              ],
+              "doc": {
+                "summary": "Transaction's maxFeePerGas is lower than the current gas price in anvil-zksync.",
+                "description": [
+                  "This error occurs when the maximum fee per gas specified in the transaction is lower than the current gas price set in the anvil-zksync node.",
+                  "To be considered valid, transactions must specify a maxFeePerGas that is greater or equal to the current gas price",
+                  "",
+                  "In anvil-zksync, the gas price can be configured when starting the node using `--l1-gas-price` argument or can be modified dynamically. By default, the node simulates a gas price model similar to the real ZKSync network, including:",
+                  "1. A base computation fee (similar to Ethereum's base fee)",
+                  "2. A simulated pubdata posting fee",
+                  "",
+                  "Even though anvil-zksync is a testing environment, it enforces these gas price validations to ensure that your tests accurately reflect how transactions would behave on the actual ZKSync network."
```

Comment:
I think you added "Note: When anvil-zksync is in forking mode, it lock-in and uses gas price at the forked block" to the two sections above by mistake when it was meant to be here